tests/lib/tenstorrent/bh_arc/src/msgqueue.c

```diff
@@ -188,6 +188,32 @@ ZTEST(msgqueue, test_msg_type_switch_clk_scheme)
 	zassert_equal(clock_wave_value, 1U);
 }
 
+ZTEST(msgqueue, test_msg_type_debug_noc_translation)
+{
+	union request req = {0};
+	struct response rsp = {0};
+
+	req.data[0] = TT_SMC_MSG_DEBUG_NOC_TRANSLATION | (1U << 8U) /* Enable translation*/
+		      | (1U << 9U)                                  /* PCIE Instance  = 1*/
+		      | (0U << 10U)                                 /*PCIE instance override*/
+		      | (0b1001U << 16U)                            /*Bad tensix columns 0 and 3*/
+		;
+	req.data[1] = (8U << 0U)/* Bad GDDR 8 */| (0b1010U << 8U) /*skip eth 1 and 3*/;
```

Comment:
Should use `BIT(...)` here too, to improve code readability
```suggestion
	req.data[1] = (8 /* Bad GDDR 8 */ | ((BIT(1) | BIT(3)) << 8) /*skip eth 1 and 3*/;
```